codalab/common.py

```diff
@@ -101,10 +101,12 @@ class State(object):
     RUNNING = 'running'   # Actually running
     READY = 'ready'       # Done running and succeeded
     FAILED = 'failed'     # Done running and failed
+    KILLED = 'killed'     # Killed by user
+    WORKER_OFFLINE = 'worker_offline'
```

Comment:
Maybe we should have a state diagram showing the possible transitions between all these states..